src/app/api/_utils/database.ts

```diff
@@ -236,8 +242,8 @@ export class PostgresClient {
     `);
   }
 
-  public async getOrganization(team_id: string): Promise<Organization> {
-    const queryString = `SELECT * FROM ${this._organizationsTable} WHERE team_id = $1`;
+  public static async getOrganization(team_id: string): Promise<Organization> {
+    const queryString = 'SELECT * FROM organizations WHERE team_id = $1';
```

Comment:
makes it a static for easier usage, and removes the organization table from here, it will always be organizations